app/views/posts/new.html.haml

```diff
@@ -0,0 +1,9 @@
+.container{:style => "font-family: Open Sans;"}
```

Comment:
Hash attribute should start with one space after the opening brace<br>Hash attribute should end with one space before the closing brace